components/scripts/gradle/03-validate-local-build-caching-different-locations.sh

```diff
@@ -346,5 +338,39 @@ EOF
   print_wizard_text "${text}"
 }
 
+explain_summary() {
+  local text
+  if [[ "${build_scan_publishing_mode}" == "on" ]]; then
+    IFS='' read -r -d '' text <<EOF
+The ‘Summary’ section below captures the configuration of the experiment and the
+two build scans that were published as part of running the experiment. The build
+scan of the second build is particularly interesting since this is where you can
+inspect what tasks were not leveraging the local build cache.
+EOF
+  else
+    IFS='' read -r -d '' text <<EOF
+The ‘Summary’ section below captures the configuration of the experiment and the
+two build scans that were published as part of running the experiment.
```

Comment:
This should be reworded since no Build Scan is published.

Reply:
Oh right. Good catch.